co-authors-plus.php

```diff
@@ -1644,8 +1644,28 @@ public function filter_author_archive_title( $title ) {
 		$author = $this->get_coauthor_by( 'user_nicename', $author_slug );
 		
 		return sprintf( __( 'Author: %s' ), $author->display_name );
+	}	
+
+	/**
+	 * Get the post count for the guest author
+	 *
+	 * @global oebject $coauthors_plus
+	 * @param object $item guest-author object.
+	 * @return int post count for the guest author
+	 */
+	public function get_guest_author_post_count( $item ) {
+		$term       = $this->get_author_term( $item );
```

Comment:
What happens if `get_author_term()` returns `NULL` from passing in an invalid variable type?

Reply:
It'll be caught by the check on line number 1661. Since that'll fail, line 1664 will return 0.

Reply:
We could bail early or throw a `WP_Error` - `missing-guest-author`, if the parameter `$item` (renaming to `guest_author`) isn't an object. Is that what you had in mind?

Reply:
> It'll be caught by the check on line number 1661. Since that'll fail, line 1664 will return 0.

That's correct, however, it will throw an error though because it will try to grab the non-existent properties of `user_nicename`.  What do you think is the best way to go about?  What are your thoughts in returning `false`?